src/main/java/org/broadinstitute/hellbender/utils/nio/PathLineIterator.java

```diff
@@ -30,10 +32,14 @@
      * @param path path to a text file.
      * @throws UserException if we cannot open the file for reading.
      */
-    public PathLineIterator(Path path) throws UserException {
+    public PathLineIterator(final Path path) throws UserException {
```

Comment:
Do we need `throws UserException`, since that is a runtime exception?

Reply:
It does not.  I'll pull it out.